client/modules/core/dashboard/dashboard.ts

```diff
@@ -0,0 +1,148 @@
+import {
+    Component,
+    OnInit,
+    OnDestroy
+} from '@angular/core';
+import {
+    Observable
+} from 'rxjs/Observable';
+import {
+    Subscription
+} from 'rxjs/Subscription';
+import {
+    MeteorObservable
+} from 'meteor-rxjs';
+import {
+    Mongo
+} from 'meteor/mongo';
+import {
+    Meteor
+} from 'meteor/meteor';
+import * as moment from 'moment';
+import template from './dashboardtemplate.html';
+import {
+    Graphdata
+} from '../../../../both/collections/csvdata.collection';
+
+@Component({
+    selector: 'dashboard',
+    template
+})
+
+export class DashboardComponent implements OnInit, OnDestroy {
+
+    current_year_header: any;
```

Comment:
if admin user logins, he should automatically go to dashboard.

if accounts or guest user logins he should go to transactions page